### PR TITLE
Added archive creation, fixed minor code style issues

### PR DESCRIPTION
**Pull request by ubuntolog**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,9 @@ clean:
 	rm $(JSBUNDLE) $(JSBUNDLE).map
 	rm -r $(WEBUI)/node_modules
 
+pack:
```
Comment (emanueldima) [inline Makefile]: This should be something like this (to be tested):
````
pack: build
        go build -o ./build/gefserver ./gefserver
        cp gefserver/config.json ./build/
        cp -r webui/app build/webui/app
        tar -cvzf gef-0.2.0.tar.gz build/*
````

Comment (ubuntolog) [inline Makefile]: Yes, I've realised that my code does not do what we need

**Comment by ubuntolog** [comment]
Go-sqlite library has some C code in it, make build was failing for this reason. Now we build a linux binary in a Docker container

File: Makefile [hunk 2]
```diff
@@ -22,6 +22,16 @@ clean:
 	rm $(JSBUNDLE) $(JSBUNDLE).map
 	rm -r $(WEBUI)/node_modules
 
+pack: dependencies webui
+	go vet ./...
```
Comment (emanueldima) [inline Makefile]: not needed

File: Makefile [hunk 3]
```diff
@@ -22,6 +22,16 @@ clean:
...
+	mkdir -p ./build/webui
+	docker run --rm -v $(GOPATH):/go -w /go/src/github.com/EUDAT-GEF/GEF golang:latest go build ./...
```
Comment (emanueldima) [inline Makefile]: not needed?

File: Makefile [hunk 4]
```diff
@@ -22,6 +22,16 @@ clean:
...
+	docker run --rm -v $(GOPATH):/go -w /go/src/github.com/EUDAT-GEF/GEF golang:latest go build -o ./build/gefserver ./gefserver
+	cp gefserver/config.json ./build/
+	cp -r webui/app build/webui/app
+	tar -cvzf gef-0.2.0.tar.gz build/* services/_internal/* ssl/*
```
Comment (emanueldima) [inline Makefile]: does this break the expected path of internal services and ssl files?

Comment (ubuntolog) [inline Makefile]: no, it does not

File: Makefile (hunk 2 above)
Comment (ubuntolog) [inline Makefile]: agree

**Pull request merged**